core/tauri/src/app.rs

```diff
@@ -406,6 +410,7 @@ pub struct App<R: Runtime> {
   setup: Option<SetupHook<R>>,
   manager: Arc<AppManager<R>>,
   handle: AppHandle<R>,
+  ran_setup: AtomicBool,
```

Comment:
shouldn't a simple bool be enough here?

Reply:
ooooooooooooooooooooooooooooooooooooooooooo